servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered.glsl

```diff
@@ -650,6 +650,7 @@ layout(constant_id = 9) const uint sc_directional_penumbra_shadow_samples = 4;
 
 layout(constant_id = 10) const bool sc_decal_use_mipmaps = true;
 layout(constant_id = 11) const bool sc_projector_use_mipmaps = true;
+layout(constant_id = 12) const bool sc_disable_fog_depth = false;
```

Comment:
Same comment as for the compatibility renderer, there is no need to have both a spec constant and a runtime check. I would reverse the naming of the spec constant to `sc_use_depth_fog` and then only check it.